extensions/registration-service-api/build.gradle.kts

```diff
@@ -10,17 +10,12 @@ dependencies {
     implementation(libs.edc.ext.http)
     implementation(libs.edc.ext.identity.did.crypto)
 
-    implementation(libs.opentelemetry.annotations)
-
     testImplementation(testFixtures(project(":spi:registration-service-spi")))
     testImplementation(testFixtures(project(":core:registration-service-client")))
 }
 
-publishing {
-    publications {
-        create<MavenPublication>(project.name) {
-            from(components["java"])
-        }
+edcBuild {
+    swagger {
+        apiGroup.set("management-api")
```

Comment:
i'm not a 100% sure, but I think that would overwrite the [api-doc for the connector's management api](https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.0.1-SNAPSHOT#/).

Reply:
for some reasons I thought they were supposed to be merged (???). But I'm seeing that we're not publishing this API currently so I will revert this

Reply:
I don't know if Swaggerhub supports merging at all 🤷 
maybe we can investigate this once we actually publish the RS's API.

Reply:
anyway maybe they'll stay better separate, so yes, let's postpone that